patches/services-service_manager-sandbox-win-sandbox_win.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/services/service_manager/sandbox/win/sandbox_win.cc b/services/service_manager/sandbox/win/sandbox_win.cc
+index d330dd221cb62822be845c742d32ac7fc164db96..8065f0ff7cd34a185199831fbbc19e9e36d01ea0 100644
+--- a/services/service_manager/sandbox/win/sandbox_win.cc
++++ b/services/service_manager/sandbox/win/sandbox_win.cc
+@@ -831,6 +831,7 @@ sandbox::ResultCode SandboxWin::StartSandboxedProcess(
+           service_manager::switches::kNoSandbox)) {
+     base::LaunchOptions options;
+     options.handles_to_inherit = handles_to_inherit;
++    options.start_hidden = true;
```

Comment:
are there any other process types started here which we don't want hidden?

Reply:
for now, we only want to hide utility process type for audio service and tor launcher service specifically

Reply:
seems like all child process on Window will enter this function but only non sandboxed child process would hit this block

Reply:
and in that case, you will see the popup terminal window so that would be only audio service and tor launcher service for now

Reply:
we can also enable `service_manager::features::kAudioServiceSandbox` and I can add this specifically for tor launcher service

Reply:
Audio Service part is updated in the issue.
https://github.com/brave/brave-browser/issues/714#issuecomment-420779221